user-manager/src/main/java/com/qpay/usermanager/mapper/UserMapper.java

```diff
@@ -0,0 +1,44 @@
+package com.qpay.usermanager.mapper;
+
+import com.qpay.libs.models.UserType;
+import com.qpay.usermanager.model.dto.authuser.UserCredentialsCreation;
+import com.qpay.usermanager.model.dto.authuser.UserCredentialsModification;
+import com.qpay.usermanager.model.dto.customer.CustomerCreation;
+import com.qpay.usermanager.model.dto.customer.CustomerModification;
+import com.qpay.usermanager.model.dto.merchant.MerchantCreation;
+import com.qpay.usermanager.model.dto.merchant.MerchantModification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
```

Comment:
you are not working here with user. It is user credentials. You should rename in auth manager respectively naming